plugins/balloontoolbar/plugin.js

```diff
@@ -680,6 +680,32 @@
 				this._detachListeners();
 			};
 
+			/**
+			 * Updates status of passed element.
+			 *
+			 * @since 4.9.0
+			 * @private
+			 * @param {CKEDITOR.ui.button/CKEDITOR.ui.richCombo}
```

Comment:
Missing param name, should be `@param {CKEDITOR.ui.button/CKEDITOR.ui.richCombo} item`.